requirements.txt

```diff
@@ -0,0 +1,13 @@
+Beaker==1.11.0
+beaker-redis==1.1.0
+Paste==3.5.0
+PasteDeploy==2.1.1
+PasteScript==3.2.1
```

Comment:
Do we want to pin versions so specifically? Might be better to allow patch or even minor upgrades depending on if they follow semver.

Reply:
Good question... the `pip freeze` strategy I used to generate these files just fixed these versions.